docs/src/main/paradox/features/organizing-pages.md

```diff
@@ -1,5 +1,18 @@
 Organizing pages
-----------------
+================
+
+### titles
+
+In your markdown files, you can specify titles by using `#` characters. Less there are `#`, bigger the title is. You can also specify titles by using `=` and `-` respectively giving the same result than `#` and `##`. For example,
+```markdown
+myTitle
+-------
+```
+is equivalent to
+```markdown
+## myTitle
+```
+and similarly for `=` and `#`.
```

Comment:
Remove description of headers. And mention that the first header becomes the title of the page.